lib/elixir/lib/module.ex

```diff
@@ -899,13 +899,16 @@ defmodule Module do
   # Otherwise, returns a generic guess
   defp merge_signature({_, meta, _}, _newer, i), do: {:"arg#{i}", meta, Elixir}
 
+  @extra_error_msg_defines? "Use Kernel.function_exported?/3 and Kernel.macro_exported?/3 " <>
+                              "to check for public functions and macros instead"
```

Comment:
Let's move to the top of the file as it is a bit unusual to have such attributes in the middle of the code? Unsure...

Reply:
I initially put them there and changed my mind.
I'll move them back

Reply:
fixed